### PR TITLE
A proposition

### PR DESCRIPTION
**Pull request by dabej**
1. Added columns for comment and boolean(main item) in Table "items", to make it more standardized when working with the functions for updating the menu by pizzabagaren.
2. Created a JSON-file with the current menu-items and prepared the Table "items" at activation of the plugin from this JSON. 
My (provocative) proposition is that we can see the Admin-page as an additional feature, and need not really be in the MVP. Maybe just write a function that allows the pizzabagare change menu items by editing(or letting someone else edit) the json-file...

### CODE REVIEW

File: pizzakit-plugin/includes/pizzakit-activator.php [hunk 1]
```diff
@@ -1,67 +1,71 @@
...
+			FOREIGN KEY (item) REFERENCES wp_items(name)
+		  )';
 
-      dbDelta($sql);
-      add_option('entries_version','1.0');
-    }
+		  dbDelta($sql);
+		  add_option('entries_version','1.0');
+		}
+		Pizzakit::add_menu_items_at_start(ABSPATH . 'wp-content/plugins/pizzakit-plugin/items_for_sale.json');
```
Comment (89netraM) [inline pizzakit-plugin/includes/pizzakit-activator.php:68]:
```suggestion
		Pizzakit::add_menu_items_at_start(plugin_dir_path(__FILE__) . '../items_for_sale.json');
```

I suggest using relative paths to files in our plugins folder. Our plugins folder might not necessarily be called `pizzakit-plugin`.

**Comment by alf-arv** [comment]
Just a nitpick, but were variables intended to be private not supposed to be named starting with underscores? I saw you removed those. Ex: $_table was changed to $table.

It's a good start for the MVP to just edit the json-file, and have the dashboard be a future add on. If so, the program would benefit from a function that updates the database tables while the site is running. Pizzakit::refresh_menu_items() for example, that deletes the items missing from the file and adds the new ones.

**Comment by 89netraM** [comment]
@alf-arv Underscores in the beginning of variable names is often used in programming languages where private fields are unavailable (like in JavaScript). `$table` is a local variable so it wouldn't been affected either way.

**Comment by alf-arv** [comment]
All right (y)

**Comment by dabej** [comment]
Have made changes and cleaned up code in new commit.

File: pizzakit-plugin/includes/items_for_sale.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
 		{"name" : "Parmaskinka", "price" : 20, "comment" : "5 skivor"},
 		{"name" : "Prosciutto Cotto (kokt skinka)", "price" : 20, "comment" : "5 skivor"},
 		{"name" : "Buffelmozzarella", "price" : 25, "comment" : "125 g"},
+		{"name" : "Halv böj", "price" : 25, "comment" : "40%"},
```
Comment (89netraM) [inline pizzakit-plugin/includes/items_for_sale.json:12]: What's this?

Comment (dabej) [reply pizzakit-plugin/includes/items_for_sale.json]: test.. removed and recommitted

**Comment by 89netraM** [comment]
I think one great thing about git is that we can delete code without worrying that we might need it in the future. And therefore I think this is acceptable.

However, would have preferred it if the removal of those functions were in a separate commit. That way, when there're needed again, we could just have reverted the commit that removed them. But it's almost as easy to just go back an copy the code.

**Comment by dabej** [comment]
I see what you mean about the add/remove-functions. But if it is as easy as going back and copying in the code, maybe it's better to keep the master-branch as "deffad" as possible. Should I resolve the conflicts and merge to master?

**Pull request merged**